types/mapbox-gl/index.d.ts

```diff
@@ -205,6 +205,14 @@ declare namespace mapboxgl {
 
         getMaxZoom(): number;
 
+        setMinPitch(minPitch?: number): this;
```

Comment:
According to https://docs.mapbox.com/mapbox-gl-js/api/#map#setminpitch "null" is also allowed here,